easybuild/easyconfigs/c/CMake/CMake-3.0.0-intel-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.0.0'
```

Comment:
this PR only includes existing easyconfigs with a new toolchain, so not touching versions of (build)dependencies